src/tests/ftest/osa/online_drain.py

```diff
@@ -203,6 +200,5 @@ def test_osa_online_drain_with_multiple_ranks(self):
         :avocado: tags=osa,osa_drain,online_drain
         :avocado: tags=OSAOnlineDrain,test_osa_online_drain_with_multiple_ranks
         """
-        self.log.info("Online Drain : Test with multiple ranks")
-        multiple_ranks = self.params.get("rank_list", '/run/multiple_ranks/*')
-        self.run_online_drain_test(1, multiple_ranks=multiple_ranks)
+        self.log.info("Offline Drain : Test with multiple ranks")
```

Comment:
Is this `Offline Drain`? The file name and the method name both have `online_drain`. The next line is `self.run_online_drain_test`.

Reply:
Done.